theano/tensor/var.py

```diff
@@ -501,6 +501,8 @@ def check_bool(args_el):
             args[ellipsis_at: ellipsis_at + 1] = (
                 [slice(None)] * (self.ndim - (len(args) - 1 - new_axes)))
 
+        args = tuple([numpy.array(inp, dtype=numpy.int64)
```

Comment:
Just add a comment to help future reader of this code understand. Something like:

Force int64 for empty list and tuple as otherwise, then end up as real numbers.